### PR TITLE
Feat/account unification UI styling

### PR DESCRIPTION
**Pull request by ayumitk**
**Pull Request Summary**

> Describe what changes this pull request makes to the repository

**Check list**

- [ ] contains breaking changes
- [ ] adds new feature
- [ ] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

**This pull request makes the following changes:**

**Adds**

- (ex: Add feature A)

**Fixes**

- (ex: Fix validation function)

**Changes**

- (ex: Change document B)

**To-dos**

> \*Feel free to remove this section if it's not applicable

- [ ] (ex: add user list)

### CODE REVIEW

**Comment by Kahonnohak** [comment]
Hi @ayumitk Thank you, I'm just leaving this for now and I'll talk to you later
Screenshot 2023-09-14 at 14 25 03

**Comment by ayumitk** [comment]
> Hi @ayumitk Thank you, I'm just leaving this for now and I'll talk to you later Sketch-2

@Kahonnohak I fixed some icon and space sizes. Also, I want to know more details about the attached image section in our next sync!
Screenshot 2023-09-18 at 5 32 27 PM

File: src/components/assets/Erc20Currency.vue [hunk 1]
```diff
@@ -1,102 +1,209 @@
...
-            <router-link :to="buildTransferPageLink(token.symbol)">
-              <button class="btn btn--sm">
-                {{ $t('assets.transfer') }}
+        </button>
+        <q-tooltip>
+          <span class="text--tooltip">{{ $t('remove') }}</span>
+        </q-tooltip>
+      </div>
+      <div>
+        <!-- TODO: the icon color will change by class name
```
Comment (ayumitk) [inline src/components/assets/Erc20Currency.vue:107]: Hi @bobo-k2, I left this kind of comment in my code. For example, this comment is for the "Add to / remove from favourite" feature. Let me know if this works for you.

Screenshot 2023-09-18 at 5 44 19 PM

**Comment by Kahonnohak** [comment]
Hi @ayumitk sorry to leave this behind, since AU is now separated that was prioritised.

Could you create the staging link again please (its expired)?

**Comment by ayumitk** [comment]
> Hi @ayumitk sorry to leave this behind, since AU is now separated that was prioritised.
> 
> Could you create the staging link again please (its expired)?

@Kahonnohak Yes, I'm thinking of creating separate PRs because this PR changes a lot, and it is hard to review all files. I'll create the following PR today and tomorrow:

- Dropdown mobile menu
- New assets page layout

Could you wait for a day to review?

**Comment by Kahonnohak** [comment]
Of course thanks Ayumi!

**Comment by Kahonnohak** [comment]
If you can fist work on the asset page that would be great x

**Pull request closed without merging**